docs/btc-methods.md

```diff
@@ -0,0 +1,73 @@
+# BTC Methods
+
+Here we document the Bitcoin methods that an account Snap may implement to
+support requests originated from dapps.
+
+## btc_sendmany
+
+This method is similar to the `sendmany` RPC method from Bitcoin Core, but its
+parameters are passed in an object instead of an array, and are named in
```

Comment:
Do we want to talk about the "parameters as an array" (like flatten) support here too? Maybe for now we can skip this and not support it yet?

Reply:
I think we want to skip it since we don't know if we will support it someday. I don't want to make any promises :)

Reply:
You're right! :D let's skip it for now!